react-components/src/components/Checkout/__test__/editableForm.test.js

```diff
@@ -114,63 +99,157 @@ describe('<EditableForm />', () => {
             </MockedProvider>
         );
 
-        const result = await waitForElement(() => {
-            return queryByText('Billing Information');
-        });
-
-        expect(result).not.toBeNull();
+        expect(asFragment()).toMatchSnapshot();
     });
 
-    it('renders the shipping method form if countries are loaded', async () => {
-        const { queryByText } = render(
-            <MockedProvider mocks={mocksQueryCountries} addTypename={false}>
-                <UserContextProvider>
-                    <CartProvider initialState={{}} reducerFactory={() => state => state}>
-                        <CheckoutProvider
-                            initialState={{ editing: 'shippingMethod', flowState: 'form' }}
-                            reducer={state => state}>
-                            <EditableForm />
-                        </CheckoutProvider>
-                    </CartProvider>
-                </UserContextProvider>
-            </MockedProvider>
-        );
-
-        const result = await waitForElement(() => {
-            return queryByText('Shipping Information');
+    it('should render the payments form if user is editing payments', async () => {
+        // Overrides Braintree jest error in tests
+        act(() => {
+            Element.prototype.scrollIntoView = jest.fn();
         });
+        const mockCart = {
+            is_virtual: false,
+            shipping_addresses: [
+                {
+                    city: 'asdf',
+                    company: null,
+                    country: { code: 'US', __typename: 'CartAddressCountry' },
+                    firstname: 'Alex',
+                    lastname: 'Kim',
+                    postcode: '55057',
+                    region: { code: 'CA', __typename: 'CartAddressRegion' },
+                    street: ['1231'],
+                    telephone: '123412341234',
+                    available_shipping_methods: [
+                        {
+                            method_code: 'flatrate',
+                            method_title: 'Fixed',
+                            carrier_code: 'flatrate',
+                            carrier_title: 'Flat Rate',
+                            __typename: 'AvailableShippingMethod'
+                        }
+                    ],
+                    selected_shipping_method: null,
+                    __typename: 'ShippingCartAddress'
+                }
+            ],
+            available_payment_methods: [
+                {
+                    code: 'braintree_paypal',
```

Comment:
I think you should ignore all Braintree logic here for now, since you're testing the displaying of the payment methods form.